Doc/c-api/buffer.rst

```diff
@@ -42,9 +42,9 @@ Simple objects such as :class:`bytes` and :class:`bytearray` expose their
 underlying buffer in byte-oriented form.  Other forms are possible; for example,
 the elements exposed by an :class:`array.array` can be multi-byte values.
 
-An example consumer of the buffer interface is the :meth:`~io.BufferedIOBase.write`
+An example consumer of the buffer interface is the :meth:`!write`
```

Comment:
This makes it lose the hyperlink. Same for the other `write` below.

Reply:
I'm pretty sure thta I saw a warning on this line, but now I have doubt since https://docs.python.org/dev/library/io.html#io.BufferedIOBase.write exists.

Reply:
I confirm that write() is a link in https://docs.python.org/dev/c-api/buffer.html I will revert this change.

Reply:
Oh, now I recall, I wanted to fix ``:meth:`write` `` below and then fixed a line which was fine :-) I updated my PR to fix both lines :-)